src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1845,15 +1845,34 @@ namespace cryptonote
     return true;
   }
   //-----------------------------------------------------------------------------------------------
+  std::string core::wrap_uptime_proof(const NOTIFY_UPTIME_PROOF::request &proof)
+  {
+    lokimq::bt_dict wrapped_bt_proof{
+      {"proof", m_service_node_list.serialize_uptime_proof(proof)},
```

Comment:
As per comments: instead of a serialized value, return (and nest) a bt_dict object here.